### ISSUE TITLE
WAVE Alert (Level A and Level AA) - Skipped heading level because a survey title uses H5 and survey description uses H3 - The H4 tag is missing

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7440]
User Issue: T15745 - Form description is a heading element?
https://surveyjs.answerdesk.io/internal/ticket/details/T15745
***
A survey title is rendered within an H5 tag, while a survey description is rendered by H3. WAVE alerts this error: Skipped heading level.

![image](https://github.com/surveyjs/survey-library/assets/22372972/2bc27aba-cb92-4d29-8896-0b4e00c575ca)

More details about the alert:

What It Means
A heading level is skipped.
Why It Matters
Headings provide document structure and facilitate keyboard navigation by users of assistive technology. These users may be confused or experience difficulty navigating when heading levels are skipped.
How to Fix It
Restructure the document headings to ensure that heading levels are not skipped.
The Algorithm... in English
A heading level is skipped (e.g., an <h1> is followed by an <h3>, with no intermediate <h2>). Note that an <h1> is not required to be the first heading within the document.
Standards and Guidelines
[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.1 Bypass Blocks (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.1)
[2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)
[Icon index](chrome-extension://jbbplnpkjmmeebjpijfedlgcdilocofh/sidebar.html#)
